wpa_supplicant/wpa_cli_zephyr.c

```diff
@@ -0,0 +1,3785 @@
+/*
```

Comment:
This huge file should not be copied from `wpa_cli.c` - better find a way to make the original work.

Reply:
We cannot re-use the file as it was intended to be a separate application with interactive usage, we could split the wpa_cli commands in to a common file and then re-use but that would be make maintaining it difficult which is why I forked it.

Reply:
For now I have not split the common code in to a separate file, to keep upmerging simple.

Reply:
I have bitten the bullet and did this.